apps/transport/lib/transport_web/controllers/explore_controller.ex

```diff
@@ -2,11 +2,13 @@ defmodule TransportWeb.ExploreController do
   use TransportWeb, :controller
 
   def index(conn, _params) do
+    consolidated_datasets_assigns =
+      Transport.ConsolidatedDataset.geo_data_datasets()
+      |> Enum.map(&{String.to_atom("#{&1}_dataset"), Transport.ConsolidatedDataset.dataset(&1)})
+      |> Map.new()
```

Comment:
```suggestion
      |> Map.new(&{String.to_atom("#{&1}_dataset"), Transport.ConsolidatedDataset.dataset(&1)})
```
Me semble que c'est équivalent